inputs/simple-list.vue

```diff
@@ -79,22 +80,26 @@
             :currentItem="currentItem"
             :disabled="isDisabled || disabled"
             :isInitialFocus="initialFocus === name"
+            :restrictItemCreation="args.restrictItemCreation"
             @add="addItem"
             @select="selectItem"
             @focus="onFocus"
             @blur="onBlur"
             @resize="onResize"
+            @triggerRestrictionError="onTriggerRestrictionError"
+            @click.native="disableRestrictionError"
+            @keydown.delete.native="disableRestrictionError"
             v-dynamic-events="customEvents"></simple-list-input>
         </div>
       </label>
-
       <div class="ui-textbox__feedback" v-if="hasFeedback || maxLength">
         <div class="ui-textbox__feedback-text" v-if="showError">{{ error }}</div>
         <div class="ui-textbox__feedback-text" v-else-if="showHelp">{{ args.help }}</div>
         <div class="ui-textbox__counter" v-if="maxLength">
             {{ valueLength + '/' + maxLength }}
         </div>
       </div>
+      <div class="restriction-error-message" v-if="showRestrictionError">{{ args.restrictedErrrorMessage }}</div>
```

Comment:
```suggestion
      <div class="restriction-error-message" v-if="showRestrictionError">{{ args.restrictedErrorMessage }}</div>
```
Also, should you add this property to the docs too?